src/Aquifer.Jobs/Services/TranslationProcessors/FrenchTranslationProcessingService.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Text.RegularExpressions;
+using Aquifer.Common.Utilities;
+
+namespace Aquifer.Jobs.Services.TranslationProcessors;
+
+public sealed partial class FrenchTranslationProcessingService : ILanguageSpecificTranslationProcessingService
+{
+    private const char NarrowNonBreakingSpace = '\u202f';
+    private const char NonBreakingSpace = '\u00a0';
```

Comment:
Changing these to `string`s would save doing a bunch of unnecessary string interpolation below.  They are only consumed as `string`s, anyway.

Reply:
There are a couple `Replace` that are using them as `char`, but you're right that overall it's causing more hassle.